demo/index.html

```diff
@@ -5,12 +5,15 @@
     <title>SQLite Wasm Demo</title>
     <script type="module" src="script.js"></script>
     <script type="module" src="main-thread.js"></script>
+    <script type="module" src="sqlite.client.js"></script>
   </head>
   <body>
     <h1>SQLite Wasm Demo</h1>
     <h2>Main thread</h2>
     <div class="main-thread"></div>
     <h2>Worker</h2>
     <div class="worker"></div>
+    <h2>Built-in Sqlite Client</h2>
```

Comment:
```suggestion
    <h2>Built-in SQLite Client</h2>
```

Reply:
Actually, maybe hint at what this does under the hood. Maybe get inspiration from Comlink’s README.